src/mono/wasm/runtime/invoke-js.ts

```diff
@@ -9,7 +10,7 @@ import { get_signature_argument_count, bound_js_function_symbol, get_sig, get_si
 import { setI32, setI32_unchecked, receiveWorkerHeapViews } from "./memory";
 import { monoStringToString, stringToMonoStringRoot } from "./strings";
 import { MonoObject, MonoObjectRef, MonoString, MonoStringRef, JSFunctionSignature, JSMarshalerArguments, WasmRoot, BoundMarshalerToJs, JSFnHandle, BoundMarshalerToCs, JSHandle, MarshalerType } from "./types/internal";
-import { Int32Ptr } from "./types/emscripten";
+import { CharPtr, Int32Ptr } from "./types/emscripten";
```

Comment:
Can additional TS imports be on separate lines to reduce conflicts, or does the language prohibit that?
```
import { Int32Ptr } from "./types/emscripten";
import { CharPtr } from "./types/emscripten";
```

Reply:
Good idea. It works. Thanks!